frontend/public/components/build-config.tsx

```diff
@@ -213,13 +214,20 @@ export const BuildConfigsPage: React.FC<BuildConfigsPageProps> = (props) => {
       items: allStrategies,
     },
   ];
+
+  const namespace = props.match.params.ns;
```

Comment:
You will get the namespace here.

```suggestion
  const namespace = props?.namespace;
```